lumicks/pylake/file_download.py

```diff
@@ -121,8 +121,7 @@ def download_from_doi(doi, target_path="", force_download=False, show_progress=T
     for file in record_metadata["files"]:
         file_name = file["filename"]
         url = (
-            f"https://zenodo.org/api/records/{record_metadata['record_id']}"
-            f"/files/{file_name}/content"
+            f"https://zenodo.org/api/records/{record_metadata['id']}" f"/files/{file_name}/content"
         )
```

Comment:
It fits on a single line now, so you could just do:
```python
url = f"https://zenodo.org/api/records/{record_metadata['id']}/files/{file_name}/content"
```

Reply:
`black` disagrees 🙂